src/styles.css

```diff
@@ -376,9 +376,11 @@ h3 {
   position: fixed;
   justify-content: center;
   align-items: center;
-  top: 0;
-  right: 0;
+  left: 50%;
+  top: 50%;
```

Comment:
Also, if we're going for reduced width: how do you feel about reduced height as well? in 75% or 80% range? If it looks good to you.